src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/BookingFacade.kt

```diff
@@ -44,4 +77,14 @@ class BookingFacade(
     // Agreed with Tim will be using a ContactService to pull back the necessary contact information related to the booking just created.
     // Multiple contacts will result in multiple emails i.e. one email per contact.
   }
+
+  private fun Collection<BookingContact>.allContactsWithAnEmailAddress() = filter { it.email != null }
+
+  private fun Collection<PrisonAppointment>.courtAppointments() = Triple(pre(), main(), post())
+
```

Comment:
This seems a little confusing as they are prison appointments, not court appointments. The only one involving the court (really) is the main hearing. Probably just terminology of calling them courtAppointments?

Reply:
Changed.